evap/staff/templates/staff_course_comments_quick.html

```diff
@@ -0,0 +1,285 @@
+            <div class="card slider-item{% if hide_reviewed and answer.is_reviewed %} seen{% endif %}{% if start %} start{% endif %}" data-id="{{ answer.id }}"{% if contributor %} data-contribution="yes"{% endif %}{% if answer.is_reviewed %} data-review="{% if answer.is_published %}publish{% elif answer.is_private %}make_private{% else %}hide{% endif %}"{% endif %}>
+                <div class="card-header">
+                    {{ questionnaire.public_name }}{% if contributor %}: {{ contributor.full_name }}{% if is_responsible %} ({% trans 'responsible' %}){% endif %}{% if label %} &ndash; <i>{{ label }}</i>{% endif %}{% endif %}
+                </div>
+                <div class="card-body">
+                    {{ question.text }}
+                    <hr />
+                {% if answer.reviewed_answer %}
+                    <p>{{ answer.reviewed_answer|linebreaksbr }}</p>
+                    <p class="comment-original">({{ answer.original_answer|linebreaksbr }})</p>
+                {% else %}
+                    <p>{{ answer.original_answer|linebreaksbr }}</p>
+                {% endif %}
+                </div>
+            </div>
+        {% endfor %}
+            <div class="alert alert-warning slider-item text-center">
+                <p>
+                    {% trans 'Some comments are still unreviewed.' %}
+                </p>
+            </div>
+            <div class="alert alert-success slider-item text-center">
+                <p>
+                    {% trans 'You have reviewed all comments on this course.' %}
+                </p>
+            {% if next_course %}
+                <p>
+                    The course {{ next_course.name }} has got {{ next_course.num_unreviewed_textanswers }} unreviewed text answers.
+                </p>
+            {% endif %}
+            </div>
+        </div>
+        <div class="card-footer d-flex justify-content-between">
+            <div class="w-25">
+                <span class="display-reviewed-counter text-muted">reviewed</span>,
+                <span class="display-unreviewed-counter text-muted">unreviewed</span>
+            </div>
+
+            <div>
+                <div data-action-set="reviewing">
+                    <button role="button" data-action="publish" class="btn btn-outline-secondary">yes</button>
+                    <button role="button" data-action="make_private" class="btn btn-outline-secondary" disabled="">private</button>
+                    <button role="button" data-action="hide" class="btn btn-outline-secondary">no</button>
+                </div>
+                <div data-action-set="incomplete" class="d-none">
+                    <button role="button" data-action="repeat-unreviewed" class="btn btn-primary">Review unreviewed</button>
+                    <button role="button" data-action="repeat-start" class="btn btn-outline-primary">Review seen</button>
```

Comment:
i'm not sure what this button is supposed to do. a button labeled `Show all again` and doing that would be fitting i guess.